client/src/shared/icons/DoublePreviousIcon.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+interface Props {
+  height: number;
+  width: number;
+  className?: string;
+}
+
+const DoublePreviousIcon: React.FC<Props> = ({
+  height,
+  width,
+  className
+}: Props) => {
+  return (
+    <svg
+      height={height}
+      width={width}
+      xmlns="http://www.w3.org/2000/svg"
+      fill="none"
+      viewBox="0 0 24 24"
+      strokeWidth={1.5}
+      stroke="currentColor"
+      className={className}
+    >
```

Comment:
```suggestion
interface Props {
  className?: string;
}

const DoublePreviousIcon: React.FC<Props> = ({
  className
}: Props) => {
  return (
    <svg
      height='24'
      width='24'
      xmlns="http://www.w3.org/2000/svg"
      fill="none"
      viewBox="0 0 24 24"
      strokeWidth={1.5}
      stroke="currentColor"
      className={className}
    >
```
Most of the time the size of an icon is the same across the design, so please give a default height and width in svg itself based on the design instead, and pass only className prop so we can update the size or add class only if we need to.
Please apply all related